wiki/HistoricalTimeline.md

```diff
@@ -1,46 +1,54 @@
 This timeline is provided mainly to assist with story writing, to make sure that all the story lines are in harmony with each other. This is the history of the galaxy as it is known to the player in the year 3013; over the course of the game the player may discover that certain things they took for historical fact turn out to be untrue, and that the history books have left out key information about the forces at work behind other events...
 
-* 2080: First human settlement on Mars.
+*2020: As Earth is ravaged by global warning, overpopulation, and war, humanity begins to look skyward. 
```

Comment:
```suggestion
*2030: As Earth is ravaged by global warning, overpopulation, and war, humanity begins to look skyward. 
```

Reply:
Well, thinking about it, this seems to say the _whole_ Earth is being, so I think 2030 or later should be fine.

Reply:
I would personally rather just not include anything this early on the timeline. I think it was a deliberate choice by MZ to leave the near future (from the present day) intentionally vague, and I think there are good reasons for doing so.